x-pack/plugins/security_solution/public/overview/components/detection_response/alerts_by_status/translations.ts

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
I created the translation file in the parent folder (`...overview/components/detection_response/`) so we can share the translations between all sections, since a lot of them will be duplicated

Reply:
ok, I'll move these there